salesagility/SuiteCRM
# Fix #9698 - Do not delete the subject when editing a tracking url from the campaign assistant

solves #9698 

## Description
A line of code is removed to solve the problem indicated in the issue.

## Motivation and Context
Prevent the information entered by the user from being lost.

## How To Test This
1. Create a campaign through the wizard.
2. In the Templates step:
- Indicate a subject in the email template.
- Create a Tracking URL.
- Edit the tracking URL and save.
3. Check that the subject has not been removed from the template.


## Types of changes


### Final checklist
